Configs/Module-Shared.xcconfig

```diff
@@ -223,3 +223,9 @@ VERSIONING_SYSTEM = apple-generic
 
 // Code will load on this and later versions of watchOS. Framework APIs that are unavailable in earlier versions will be weak-linked; your code should check for null function pointers or specific system versions before calling newer APIs.
 WATCHOS_DEPLOYMENT_TARGET = 2.0
+
+// macOS Catalyst support
+SUPPORTS_MACCATALYST = YES
+
+// macOS Catalyst support
+DERIVE_MACCATALYST_PRODUCT_BUNDLE_IDENTIFIER = NO
```

Comment:
This one is meaningless for framework, only for the Application. So we set it to `NO` (The Xcode 11 default value for framework)

Mac Catalyst App must not have the same bundle ID as iOS App. So this is import for Application to assign the different bundle ID. This options `YES` will add a `maccatalyst` prefix to the bundleID, which make the automatically sign works.